### ISSUE TITLE
HNC: kubectl plugin does not show `hierarchical-namespaces` in the Usage section

### ISSUE THREAD

**Issue by sophieliu15** [kubernetes-sigs/multi-tenancy#601]
`hierarchical-namespaces` is added in the Usage section in the [root command](https://github.com/kubernetes-sigs/multi-tenancy/blob/2946d71727d081f8be651fc86d68b948414bbabc/incubator/hnc/pkg/kubectl/root.go#L59). However, when typing `kubectl hns`, the output shows:

```
Manipulates hierarchical namespaces provided by HNC

Usage:
  kubectl [command]

Available Commands:
  config      Manipulates the HNC configuration
  create      Creates a subnamespace under the given parent.
  ...
```

Ideally, the Usage section should be `kubectl hierarchical-namespaces [command]`

This also impacts the usage section of subcommands. For example, when typing `kubectl hns create`, the usage section is:

```
Usage:
  kubectl create -n PARENT CHILD [flags]
```

Ideally, it should be `kubectl hierarchical-namespaces create -n PARENT CHILD [flags]`

P.S. We should probably also change `kubectl hierarchical-namespaces` to `kubectl hns` once we fix the issue described above.

**Comment by sophieliu15** [comment]
cc:
@adrianludwin 
@yiqigao217 
@adrianchung

**Comment by sophieliu15** [comment]
This is fixed in #615
